todo/src/terminal/error.rs

```diff
@@ -1,23 +1,22 @@
-use crate::repository::file_storage::error::StorageError;
+use crate::repository::todo::error::StorageError;
 use std::{fmt, io::Error};
+use uuid::Error as UUIDError;
 
 #[derive(Debug)]
 pub enum TerminalError {
     Stdout(Error),
     Stdin(Error),
-    ParseInt(String),
-    IndexError,
     StorageError(StorageError),
+    UUIDParse(UUIDError),
```

Comment:
A convenção de nomes do Rust diz que isso deve ser `UuidParse` e `UuidError`.

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start.

https://rust-lang.github.io/api-guidelines/naming.html